BookingsApi/BookingsApi.Contract/V1/Responses/HearingDetailsResponse.cs

```diff
@@ -49,11 +49,6 @@ public class HearingDetailsResponse
         /// </summary>
         public List<ParticipantResponse> Participants { get; set; }
         
-        /// <summary>
```

Comment:
Not sure this is within scope of the change, I dot mind personally, but do we need to update admin web if we're removing a property from the v1 response?